### PR TITLE
feat: Implement MariaDB resource

### PR DESCRIPTION
**Pull request by irmatov**

### CODE REVIEW

**Review by roneli — APPROVED** [review]
LGTM

File: resources/services/mariadb/servers.go [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+			},
+			{
+				Name:        "replica_capacity",
+				Description: "The maximum number of replicas that a master server can have.",
+				Type:        schema.TypeInt,
+				Resolver:    schema.PathResolver("ServerProperties.ReplicaCapacity"),
+			},
+			{
+				Name:        "public_network_access",
+				Description: "PublicNetworkAccess - Whether or not public network access is allowed for this server",
```
Comment (amanenk) [inline resources/services/mariadb/servers.go:135]: remove `PublicNetworkAccess -`

File: resources/services/mariadb/servers.go [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+//                                               Table Resolver Functions
+// ====================================================================================================================
+
+func fetchMariadbServers(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	svc := meta.(*client.Client).Services().MariaDB.Servers
+	r, err := svc.List(ctx)
+	if err != nil {
+		return err
+	}
+	if r.Value == nil {
```
Comment (amanenk) [inline resources/services/mariadb/servers.go:277]: nit: check if there is a pagination

**Review by amanenk — APPROVED** [review]
LGTM, a few nits

File: resources/services/mariadb/servers.go (hunk 1 above)
Comment (irmatov) [reply resources/services/mariadb/servers.go:135]: Fixed, thanks

**Pull request merged**